crates/compiler/alias_analysis/src/lib.rs

```diff
@@ -358,7 +358,9 @@ where
 
     match opt_level {
         OptLevel::Development | OptLevel::Normal => morphic_lib::solve_trivial(program),
-        OptLevel::Optimize | OptLevel::Size => morphic_lib::solve(program),
+        // TODO(#7367): Change this back to `morphic_lib::solve`.
```

Comment:
Should we keep this comment given that it is faster?

Reply:
Put a lot more detail into the todo. My gut feeling is that we want full morphic in the long run, but the bug is leading to it having worse perf than it should. I would guess that once the bug is fixed, we will be able to find cases with solid perf wins from morphic.